projects/module-2/whats-cookin-part-two-agile.md

```diff
@@ -0,0 +1,206 @@
+---
+title: What's Cookin - Part 2!
+tags: javascript, mocha, testing, fetch, agile
+---
+
+## Goals and Objectives
+
+- Work with a local server and make network requests to API endpoints to retrieve and manipulate data.
+- Ensure your app follows best practices for accessibility
+- Practice talking about your code and high level technical concepts
+- Implement feedback from a usability test to improve product
+- Improve group workflow.
+
+In this project, you will use your project from What’s Cookin’ (Part 1) and build on top of that. This is an opportunity to refactor your code, utilize agile processes, complete unfinished work, build new features, and take advantage of instructor feedback.
+
+## Timeline
+
+Dates and deliverables to be aware of:
+
+- **Tuesday of Week 4** - Project kickoff
+    - Perform your retrospective and DM your instructor project manager two things from your retrospective (see below) that you want to keep doing and two things that you want to improve on as a group heading into Part 2.
+    - DM your instructors your “definition of done” for your track.
+- **Monday of Week 5** - Project due
+- **Tuesday of Week 5** - Project evaluations (presentation and interview questions)
+    - Instead of submitting code in-depth review, you’ll be giving a presentation about your process and how you implemented your track’s features as well as answering a couple interview-style questions.
+
+## Daily Check-Ins
+
+Every work day, your group should do a check-in as a whole group, preferably live over Zoom, and discuss at least these things:
+
+- What is the state of the project so far? (update your GitHub project board if necessary)
+- What is each person working on today?
+- Are there any blockers, and what is your plan to get through the blocker?
+
+Your daily check-in schedule (sometimes called “stand-ups”) should be outline in your DTR so the whole group is aware of the meeting.
+
+## Retrospective
+
+A retrospective (retro) is a look back at the recent development period, in this case, Part 1 of the project, in order to improve future development work as a team. For your retrospective, meet with your team to discuss the following questions:
+
+- What helped you to be successful as a team?
+- What caused the problems that you had in Part 1?
+- Were daily check-ins productive and helpful? If not, what would you change?
+- What questions do we have about Part 2 of the project?
+- Is there anything we should change in our DTR?
+
+When you’ve finished your retro, DM your project manager two things from your retrospective (see below) that you want to keep doing and two things that you want to improve on as a group heading into Part 2.
+
+## Applying Instructor Feedback from Part 1
+
+- Implement instructor feedback from Part 1
```

Comment:
Maybe also include:
- If necessary, ask the instructor clarifying questions about their feedback